src/js/modules/RoutingApp.js

```diff
@@ -17,51 +22,62 @@ const RoutingApp = ({ sections, roles }) => {
     return Object.keys(sections).map(sectionSlug => {

```

Comment:
I'd appreciate a short comment that just mentions the reason why you have to use functions instead of components. I think people are getting the wrong idea about React design because of this